### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by DavidSeptimus-Klotho**
This PR upgrades out-of-date or vulnerable dependencies and removes some unused dependencies.

`@types/node` was mostly left alone since it's not entirely clear which version we should be using (though it should probably be consistent).

A `Makefile` with some rules covering common bulk actions was added to the repo root to make maintenance of all the sample apps a bit easier.

### CODE REVIEW

**Comment by DavidSeptimus-Klotho** [comment]
We may want to wait on actually merging this until the runtime dependencies are upgraded and we've had a chance to run through a couple of the upgraded samples alongside the upgraded runtime in the cloud.

File: ts-typeorm/typeorm/logic.ts [hunk 1]
```diff
@@ -10,11 +10,12 @@ export const initialize = async (): Promise<DataSource> => {
    * }
    */ 
     const AppDataSource = new DataSource({
-        type: "postgres",
-        entities: [User],
-        synchronize: true,
-        logging: false,
-    })
+     type: "sqlite",
```
Comment (ewucc) [inline ts-typeorm/typeorm/logic.ts:13]: is this related to the dependency changes?

Comment (gordon-klotho) [inline ts-typeorm/typeorm/logic.ts:13]: This change seems unrelated, but I don't mind it

**Review by gordon-klotho — APPROVED** [review]
I don't think we need to wait on runtime changes in klotho for updates here.

File: ts-typeorm/typeorm/logic.ts (hunk 1 above)
Comment (DavidSeptimus-Klotho) [reply ts-typeorm/typeorm/logic.ts:13]: This is because you need to actually have a postgresql  database running locally to actually test the app if you specify postgres.

Comment (DavidSeptimus-Klotho) [reply ts-typeorm/typeorm/logic.ts:13]: the runtime will swap out the sqlite bit with the appropriate postgres config at run-time.

**Review by ewucc — APPROVED** [review]

File: ts-typeorm/typeorm/logic.ts (hunk 1 above)
Comment (DavidSeptimus-Klotho) [reply ts-typeorm/typeorm/logic.ts:13]: true. like I mentioned in response to @ewucc, I made the change so I (and presumably anyone else who wants to run that sample) can run the sample without setting up postgres locally.
